map_merge:
  ros__parameters:
    /TB3_1/map_merge/init_pose_x: 0.0
    /TB3_1/map_merge/init_pose_y: 0.0
    /TB3_1/map_merge/init_pose_yaw: 0.0
    /TB3_1/map_merge/init_pose_z: 0.0
    /TB3_2/map_merge/init_pose_x: -0.02508533876302781
    /TB3_2/map_merge/init_pose_y: -0.02131316581374913
    /TB3_2/map_merge/init_pose_yaw: 0.0
    /TB3_2/map_merge/init_pose_z: 0.0
    known_init_poses: true
    merging_rate: 2.0
    robot_map_topic: map
    robot_namespace: TB3
    world_frame: world
